examples/end-to-end-examples/sec_10k_qa/README.md

```diff
@@ -0,0 +1,190 @@
+- Downloading and processing the data for domain specific pretraining of [MPT-7b](https://huggingface.co/mosaicml/mpt-7b) ([process and upload](#1-acquiring-the-sec-10-k-data), [MDS conversion](#2-mds-conversion))
+- Finetuning MPT-7b on financial text ([finetuning](#3-finance-finetune-mpt-7b))
+- Instruction finetuning MPT-7b on the data we used to build MPT-7b-instruct ([instruct finetuning](#4-instruct-finetune-mpt-7b))
+- Deploying your finetuned MPT-7b and an Instructor embedding model for inference using the MosaicML inference service ([convert to Hugging Face](#5-convert-the-composer-checkpoint-to-a-huggingface-checkpoint), [deploy](#6-deploy-your-model-and-an-embedding-model))
+- Using the MosaicML inference integration in LangChain to connect to your deployed models ([application](#7-application-with-gradio))
+- Building a simple frontend with [gradio](https://gradio.app/) to tie everything together ([application](#7-application-with-gradio))
+- Next steps you might take ([what's next](#what-next))
+
+At the end of this tutorial, you will have a simple web application (pictured below) that answers questions about SEC Form 10-K documents using a model that you finetuned and deployed using MosaicML! All the normal caveats about language models and making things up apply, this is a simple prototype and not production ready, and there are a variety of avenues you can pursue after going through this tutorial to improve the quality and apply the MosaicML platform to your task :)
+
+![demo](web_app_screenshot.png)
+
+
+## Which MosaicML tools will we use?
+- [LLM-foundry](https://github.com/mosaicml/llm-foundry): An open-source PyTorch library of tools for training, finetuning, evaluating, and deploying LLMs for inference.
+- [Composer](https://github.com/mosaicml/composer): An open-source PyTorch library for easy large scale deep learning.
+- [Streaming](https://github.com/mosaicml/streaming): An open-source PyTorch library for efficiently and accurately streaming data from the cloud to your training job, whereever that job is running.
+- [MCLI](https://docs.mosaicml.com/projects/mcli/en/latest/): The command line interface for running training and inference jobs on the MosaicML platform.
+
+
+## Setup
+
+All commands in this tutorial are going to be run using MCLI yamls. For understanding how this works, and what is actually running, there are a few important details to understand.
+
+1) The MosaicML platform does not have permanent storage on the compute nodes. This means that all data will be streamed in/out to/from a cloud object store. See [MosaicML setup](#mosaicml-platform-setup) for more details on setting up your cloud provider of choice.
+1) The `command` section of the yaml is what will actually get run on the compute node. If you are trying to debug/run something locally, you should run what appears in the `command` section (after setting up your local environment).
```

Comment:
I found this transition a bit harsh. We start referencing yaml's but I don't think they're really introduced above or anywhere prior to this. Do we expect the user to be familiar with MCLI yamls at this point? I see we provide a link to relevant MCLI documentation below explaining the yaml schema's below. Maybe it's just sufficient to say that "We will be working with MCLI run YAMLs, for more information please read the following links..."

Either way I think we need to anchor the yaml concept somehow otherwise users's not quite acquainted will get confused here.

Reply:
Yup, makes sense, will adjust

Reply:
I say "All commands in this tutorial are going to be run using MCLI yamls." at the start of this section, let me add a bit more verbiage at the start there.